spk/cryptsetup/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = cryptsetup
+SPK_VERS = 2.6.1
+SPK_REV = 1
+SPK_ICON = src/papirus-folder-locked.png
+
+REQUIRE_KERNEL ?= 1
+REQUIRE_KERNEL_MODULE  = CONFIG_BLK_DEV_DM_BUILTIN:drivers/md:dm-builtin
+# MD required because of Synology patches
+REQUIRE_KERNEL_MODULE += CONFIG_BLK_DEV_MD:drivers/md:md-mod
+REQUIRE_KERNEL_MODULE += CONFIG_BLK_DEV_DM:drivers/md:dm-mod
+REQUIRE_KERNEL_MODULE += CONFIG_DM_CRYPT:drivers/md:dm-crypt
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API:crypto:af_alg
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API_HASH:crypto:algif_hash
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API_SKCIPHER:crypto:algif_skcipher
```

Comment:
Interesting to see that this portion of the code is finally finding new usages.  If I recall you can use `REQUIRE_KERNEL :=  1`.